From patchwork Mon Jan  4 00:28:43 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Rosen Penev <rosenp@gmail.com>
X-Patchwork-Id: 1421977
Return-Path: 
 <openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.openwrt.org
 (client-ip=2001:8b0:10b:1231::1; helo=merlin.infradead.org;
 envelope-from=openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org;
 receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=fail (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 secure) header.d=lists.infradead.org header.i=@lists.infradead.org
 header.a=rsa-sha256 header.s=merlin.20170209 header.b=uQtKgS+E;
	dkim=fail reason="signature verification failed" (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=lFMmuPBx;
	dkim-atps=neutral
Received: from merlin.infradead.org (merlin.infradead.org
 [IPv6:2001:8b0:10b:1231::1])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256)
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4D8Gkj42cRz9sSC
	for <incoming@patchwork.ozlabs.org>; Mon,  4 Jan 2021 11:30:26 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=merlin.20170209; h=Sender:Content-Transfer-Encoding:
	Content-Type:List-Subscribe:List-Help:List-Post:List-Archive:List-Unsubscribe
	:List-Id:MIME-Version:Message-Id:Date:Subject:To:From:Reply-To:Cc:Content-ID:
	Content-Description:Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc
	:Resent-Message-ID:In-Reply-To:References:List-Owner;
	bh=Za0idB7a3CjtGy5Jnk7qHQfnK5TW7/N9LmNTTsTVt3k=; b=uQtKgS+EwK7nIsT+iJJC1DgWHX
	NlKp8BUgoFl9Dm7F4LJA9orgAz5LpA+pViccudyCVMvQX51NybOEzHm/ZmCv3TUxMGfHV7gR1nbZE
	wAXhlUcPdte8jq4Ry5+JgRHTZLln20A0gqhcKtvJTG0CQvvYeMzte+yrnvew7Yiqs9jV7zHRecF/+
	oCubUVz0gLl5S0dF98oZzxg2Xv9q//SRrg43lO69D7j5X3380AZQX1aJ4x2BseDjm4tW68Du90Zul
	QVu55cBA1nuD3rSNA/UTkw6PDjcIyiViC1XIEXRZ0NVbHKFbMsqP8xqKWrBtmYJOfm2W8pL7242c8
	/CEekKXA==;
Received: from localhost ([::1] helo=merlin.infradead.org)
	by merlin.infradead.org with esmtp (Exim 4.92.3 #3 (Red Hat Linux))
	id 1kwDkA-0007j8-5y; Mon, 04 Jan 2021 00:29:06 +0000
Received: from mail-pj1-x1035.google.com ([2607:f8b0:4864:20::1035])
 by merlin.infradead.org with esmtps (Exim 4.92.3 #3 (Red Hat Linux))
 id 1kwDjr-0007ZN-8h
 for openwrt-devel@lists.openwrt.org; Mon, 04 Jan 2021 00:28:50 +0000
Received: by mail-pj1-x1035.google.com with SMTP id n3so4526945pjm.1
 for <openwrt-devel@lists.openwrt.org>; Sun, 03 Jan 2021 16:28:46 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=mChXKm39GWl+vwxD6Cf2jppYeTR86TF/0JIgbzhgA1s=;
 b=lFMmuPBxry7myN18N47phhjmujrKruY4UD4mo7eLRWkrFFrov8Rn4Uqbf3ms9onQLW
 EvhrchtVdbB/3QXQyTYCG+5xL8+4nO5ycYzxJgdV+yvQ2AIkPyq4yLeuL5L6P/dIKd9B
 XpzFPNn4wPJ39kYzYmnEGKs4oRJ1fnNk8L5A2+oSS8XpY4p+vnjDnJfOofvLY4GziuwJ
 1bQmZDsDTQOSWdHeeaKOTTLwmoLrpNoINArI5++2Ur7S5O/PJ8CkwHGK2tkyhBzNWboU
 v8IUQ/5JQq8m9lWjkQiVicLOdKIEbCb8xulSEKTk1YupigSHiiSy0P9TEb0uiP53PfvF
 BXTQ==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=mChXKm39GWl+vwxD6Cf2jppYeTR86TF/0JIgbzhgA1s=;
 b=KJHxM1hfsLwoqFE7qerLEAsbKpGj5uSQfmC4sndqtUQbeDfL3aLeFoD8mU2rm/KIn/
 KC20oZv5L6BhdVWsH73+ZHt0HpZANQi6ap3ordih35MFz5UAH8SD//3dNMv0odGpTfjo
 1DmLXkUf6cMIFnPdNxM8M3k7LEcwZKFEGOLVJI7MMoG1knwbQypuFuuF2yCaw1QdsX80
 2f6Ma2AE83HCXil5oYHmPNBFS9M9E2eOiWFIaP8FSTAoDpHYfs7HLHYRugCyY/+76DsQ
 gIyi6Lt26SnX5F/yMu89JFs/iu88lSY0getyKgP0f9V9co0O9XwR/hsgM6G7F1FRf+0B
 gPPg==
X-Gm-Message-State: AOAM530w0BBFHc1ycPeUFrdsSNtmLPN+61bamugOshsMVKPZwb7p7xiK
 0XUDCzDnIqwQ7eGR0s/Q5pzHyX0wqjz8aA==
X-Google-Smtp-Source: 
 ABdhPJxuPdNmWEwU2eEHwnzHKI6gvv/KuJZeAvxTNvMzoBnuvRepEAstQRXbSXWxoJZLJyMfqJUcwQ==
X-Received: by 2002:a17:90a:b114:: with SMTP id
 z20mr26895045pjq.14.1609720124700;
 Sun, 03 Jan 2021 16:28:44 -0800 (PST)
Received: from mangix-trapnet.lan ([2001:470:1f05:79e::a89])
 by smtp.gmail.com with ESMTPSA id t7sm53641804pfe.179.2021.01.03.16.28.43
 for <openwrt-devel@lists.openwrt.org>
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Sun, 03 Jan 2021 16:28:44 -0800 (PST)
From: Rosen Penev <rosenp@gmail.com>
To: openwrt-devel@lists.openwrt.org
Subject: [PATCH] mbedtls: update to 2.16.9
Date: Sun,  3 Jan 2021 16:28:43 -0800
Message-Id: <20210104002843.763000-1-rosenp@gmail.com>
X-Mailer: git-send-email 2.29.2
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20210103_192848_815524_01749D96 
X-CRM114-Status: GOOD (  11.00  )
X-Spam-Score: -0.2 (/)
X-Spam-Report: SpamAssassin version 3.4.4 on merlin.infradead.org summary:
 Content analysis details:   (-0.2 points)
 pts rule name              description
 ---- ----------------------
 --------------------------------------------------
 -0.0 RCVD_IN_DNSWL_NONE     RBL: Sender listed at https://www.dnswl.org/,
 no trust [2607:f8b0:4864:20:0:0:0:1035 listed in]
 [list.dnswl.org]
 0.0 FREEMAIL_FROM          Sender email is commonly abused enduser mail
 provider [rosenp[at]gmail.com]
 -0.0 SPF_PASS               SPF: sender matches SPF record
 0.0 SPF_HELO_NONE          SPF: HELO does not publish an SPF Record
 0.1 DKIM_SIGNED            Message has a DKIM or DK signature,
 not necessarily
 valid
 -0.1 DKIM_VALID Message has at least one valid DKIM or DK signature
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from
 author's domain
 -0.1 DKIM_VALID_EF          Message has a valid DKIM or DK signature from
 envelope-from domain
X-BeenThere: openwrt-devel@lists.openwrt.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: OpenWrt Development List <openwrt-devel.lists.openwrt.org>
List-Unsubscribe: <https://lists.openwrt.org/mailman/options/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=unsubscribe>
List-Archive: <http://lists.openwrt.org/pipermail/openwrt-devel/>
List-Post: <mailto:openwrt-devel@lists.openwrt.org>
List-Help: <mailto:openwrt-devel-request@lists.openwrt.org?subject=help>
List-Subscribe: <https://lists.openwrt.org/mailman/listinfo/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=subscribe>
Sender: "openwrt-devel" <openwrt-devel-bounces@lists.openwrt.org>
Errors-To: 
 openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 package/libs/mbedtls/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/libs/mbedtls/Makefile b/package/libs/mbedtls/Makefile
index 27f50f8dde..0367b61a4c 100644
--- a/package/libs/mbedtls/Makefile
+++ b/package/libs/mbedtls/Makefile
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mbedtls
-PKG_VERSION:=2.16.8
+PKG_VERSION:=2.16.9
 PKG_RELEASE:=1
 PKG_USE_MIPS16:=0
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/ARMmbed/mbedtls/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=fe9e3b15c3375943bdfebbbb20dd6b4f1147b3b5d926248bd835d73247407430
+PKG_HASH:=fc17ff7d8c11d08f23ae2800a18269408ad2c24ea6bb8b9363e41a01c2425697
 
 PKG_BUILD_PARALLEL:=1
 PKG_LICENSE:=GPL-2.0-or-later
